css/jquery-ui.css

```diff
@@ -1065,36 +1065,22 @@ a.ui-button:active,
 }
 .ui-icon,
 .ui-widget-content .ui-icon {
-	background-image: url("images/ui-icons_444444_256x240.png");
-	background-image: url("../../images/css/images/ui-icons_454545_256x240.png");
+	background-image: url("../images/css/images/ui-icons_454545_256x240.png");
```

Comment:
@GregoryPost wasn't there an issue with security when we used relative pathing for assets?